"0","ventas = read.table('ventas.txt')"
"2","cannot open file 'ventas.txt': No such file or directory"
"2","Error in file(file, ""rt"") : cannot open the connection
"
